frontend/src/component/project/Project/ProjectFeatureToggles/PaginatedProjectFeatureToggles.tsx

```diff
@@ -659,18 +659,23 @@ export const PaginatedProjectFeatureToggles = ({
                             <ConditionallyRender
                                 condition={searchValue?.length > 0}
                                 show={
-                                    <TablePlaceholder>
-                                        No feature toggles found matching
-                                        &ldquo;
-                                        {searchValue}
-                                        &rdquo;
-                                    </TablePlaceholder>
+                                    <Box sx={{ padding: theme.spacing(3) }}>
```

Comment:
I think we should move `<Box sx={{ padding: theme.spacing(3) }}>` out of the render, then we can reuse it.

Reply:
Good idea